cems-api.yml

```diff
@@ -4331,6 +4331,32 @@ paths:
         404:
           description: "Company or course not found"
           
+  /company/{companyId}/providers:
+    get:
+      tags:
+        - "Company"
+      summary: "Get a list of Providers for a company"
```

Comment:
'providers for a company'

pretend you are a corporate client trying to consume our api for all purposes ... adding users, assigning content, etc .. will they know what this endpoint is and what it is for?

'providers for a company' is only clear within the context of third party authentication ... right this moment I'm not sure what the route should be ... but it should be more clear than this I think?

Reply:
maybe `/company/{id}/sso/providers` ?

Reply:
The way that I understand it, HomeCEU is consuming the API internally to provide a feature for our corporate clients so their users can log into the application using a third party provider such as Google.  I wasn't aware that corporate clients would actually be coding to our api and viewing our api documentation.  Lets discuss this so I better understand who's the user of the documentation and perhaps reword the documentation so it's more clear.

Reply:
I like Carlos's idea to add "sso" to the route.  However this does increase scope.  For consistency, we would need to apply the same change to all existing backend endpoints and add "sso."

This existing backend endpoints:
- /company/{company_id}/provider/{provider}/username-mapping/import
- /company/{company_id}/provider/{provider}/employee/{employee_id}/username-mapping
- /company/{company_id}/provider/{provider}/username-mapping
- /company/{company_id}/provider/{provider}/employee/{employee_id}/username-mapping

Would need to be changed to 
- /company/{company_id}/sso/provider/{provider}/username-mapping/import
- /company/{company_id}/sso/provider/{provider}/employee/{employee_id}/username-mapping
- /company/{company_id}/sso/provider/{provider}/username-mapping
- /company/{company_id}/sso/provider/{provider}/employee/{employee_id}/username-mapping